pkg/controllers/disruption/expiration_test.go

```diff
@@ -234,6 +235,37 @@ var _ = Describe("Expiration", func() {
 		Expect(ExpectNodes(ctx, env.Client)).To(HaveLen(0))
 		ExpectNotFound(ctx, env.Client, nodeClaim, node)
 	})
+	It("should update metrics for expired nodes", func() {
```

Comment:
nit: could consider moving this into two tests: alpha and beta, but I don't think it's too necessary since we're going to be removing it super soon.

Reply:
This raises a good question. Should we even have tests for the alpha metrics since the next version won't even have the beta version?